packages/noports_core/lib/src/sshnp/models/sshnp_arg.dart

```diff
@@ -350,18 +351,28 @@ class SshnpArg {
   );
   static const authenticateClientToRvdArg = SshnpArg(
```

Comment:
Can we add an abbreviation to these authentication flags?

Maybe -a for client auth, and -A for daemon auth?

Reply:
done